### ISSUE TITLE
No atmosphere shader at all Blender 3.4+ and PSA v1.6.2

### ISSUE THREAD

**Issue by pirzhnk** [PhysicalAddons/physical-starlight-and-atmosphere#83]
When i am enabling atmosphere the lighting is fine but the background is black
I tried it in a new file but the issue is the same
One thing that can fix it is applying a preset from asset library but i can only apply "earth", and only sun size and clouds are changing when i apply other presets, lightning is still the same as "earth"

![image](https://user-images.githubusercontent.com/125807668/219951829-8eaed9ea-50d2-4339-baae-c1533b2a85d9.png)
![image](https://user-images.githubusercontent.com/125807668/219952012-b1c886ff-f139-4e7e-b437-816ac61d7724.png)

i am using blender 3.3 and the lates version of addon
It was fine in previous version i was using

**Comment by lal0cura** [comment]
Same Issue Here.  Using Blender 3.3.3 LTS, and Ubuntu Linux 22.04.
Blender .zip downloaded from Official site.

**Comment by karlisup** [comment]
Hi @pirzhnk and @lal0cura,
Thank you for reporting the issue!
TL;DR
Bugfix that was introduced in PSA v1.6.2 breaks compatibility with older Blender versions.

Blender has introduced a new Mix node that is available on Blender 3.4, but not on earlier versions. Therefore latest addon v1.6.2 version doesn't work on versions below 3.4. 
![image](https://user-images.githubusercontent.com/3999661/220294521-db3667c7-4952-4f6b-9133-207f070134bb.png)
Although we provided info that describes which Blender version this addon version is compatible with - it is not displayed anywhere in the Blender and the user is not being warned in any way. We are looking for an alternative solution to display this warning. Meanwhile, we have pointed it out in our documentation [requirements section](https://www.physicaladdons.com/psa/getting-started/):

![image](https://user-images.githubusercontent.com/3999661/220295100-e5780641-85bf-471c-8629-9b04f83dada5.png)

Sorry for any inconvenience caused regarding this issue. If using Blender 3.3.3 LTS we suggest using the previous Addon 1.6.1 version. Let me know if this fixes your issue.

**Comment by ReinaVV** [comment]
No recent activity and no signs of the issue in newer builds.
Closing to keep the tracker tidy.
